data/regr_generator.js

```diff
@@ -0,0 +1,365 @@
+let Stream = require('./base_stream.js')
```

Comment:
Name of the file should be longer. regression_generator? Should it be rather data_generator?